sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -148,6 +154,19 @@ mobile_with_searches AS (
     unioned.submission_date,
     unioned.uri_count,
     unioned.is_default_browser,
+    SPLIT(unioned.normalized_os_version, '.')[SAFE_OFFSET(0)] AS os_version_major,
+    CONCAT(
+      SPLIT(unioned.normalized_os_version, '.')[SAFE_OFFSET(0)],
+      '.',
+      SPLIT(unioned.normalized_os_version, '.')[SAFE_OFFSET(1)]
+    ) AS os_version_minor,
```

Comment:
Is it fine that this is still gonna be a String field? Looker won't be able to do things like "filter for versions higher than x"

Reply:
Right I have that point in mind. 
My plan is to use the UDF browser_version_info. But only after we are able to solve this [issue,](https://github.com/mozilla/bigquery-etl/issues/2917). This is to avoid having a truncated datapoint (wrong data). Another option is to cast this value to number for now..